cli/util/util.go

```diff
@@ -284,6 +285,44 @@ func GetTarantoolVersion(cli *cmdcontext.CliCtx) (string, error) {
 	return cli.TarantoolVersion, nil
 }
 
+// GetTarantoolIncludePath returns and caches a directory with
+// header files supplied with tarantool.
+func GetTarantoolIncludePath(cli *cmdcontext.CliCtx, cliOpts *config.CliOpts) (string, error) {
```

Comment:
Unit test for this functions is missing.

Reply:
This function This function depends on runtime and I don't know how to correctly write unit tests for it.

Reply:
Just fill CliCtx and CliOpts with the data for test and call the function. If there is executable to be run, generate a shell script with the specified output, set TarantoolExecutable to this script and call the function.